app/Models/User.ts

```diff
@@ -0,0 +1,33 @@
+import { DateTime } from 'luxon';
+import Hash from '@ioc:Adonis/Core/Hash';
+import { column, beforeSave, BaseModel } from '@ioc:Adonis/Lucid/Orm';
+
+export default class User extends BaseModel {
+    @column({ isPrimary: true })
+    public id: number;
+
+    @column()
+    public name: string;
+
+    @column()
+    public email: string;
+
+    @column({ serializeAs: null })
+    public password: string;
+
+    @column()
+    public rememberMeToken?: string;
+
+    @column.dateTime({ autoCreate: true })
+    public createdAt: DateTime;
+
+    @column.dateTime({ autoCreate: true, autoUpdate: true })
+    public updatedAt: DateTime;
+
+    @beforeSave()
```

Comment:
ADonis sure gives you everything!
Have you used it alot before?

Reply:
Yes, I have used it before from version 4.0 to 4.1 and now on version 5 to create most of the APIs that require the usage of nodejs and typescript in the backend.

Reply:
Cool, what is the best key takeaway in your mind?

Reply:
The best key takeaway is it has evolved from versions 3 and 4 where some things were missing and development was rather complex using this framework, and now it is on course to compete with widely used frameworks like express, and from a development point, it makes development faster as it consists mostly of the packages one needs to get an application up and running in not time.

Reply:
Sounds good!